lib/cache/on-disk.ts

```diff
@@ -31,6 +31,7 @@ import type {GetResult} from '../../types/cache.interfaces.js';
 import {logger} from '../logger.js';
 
 import {BaseCache} from './base.js';
+import {v4 as uuidv4} from 'uuid';
```

Comment:
Any reason not to use the built in https://nodejs.org/docs/latest-v14.x/api/crypto.html#crypto_crypto_randomuuid_options ? Speed?

Reply:
none at all: didn't realise we had that too!